### PR TITLE
Provide an authorizer that works based on attributes

### PR DESCRIPTION
**Pull request by mmoayyed**
See javadocs and tests

### CODE REVIEW

**Comment by mmoayyed** [comment]
This may also be used partially to handle https://github.com/pac4j/pac4j/issues/633

File: pac4j-core/src/main/java/org/pac4j/core/authorization/authorizer/RequireAnyAttributeAuthorizer.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        this.valueToMatch = valueToMatch;
+    }
+
+    @Override
+    protected boolean check(final WebContext context, final U profile, final String element) throws HttpAction {
+        if (!profile.containsAttribute(element)) {
+            return false;
+        }
+
+        if (StringUtils.isEmpty(this.valueToMatch)) {
```
Comment (leleuj) [inline pac4j-core/src/main/java/org/pac4j/core/authorization/authorizer/RequireAnyAttributeAuthorizer.java]: Spring framework is only an optional dependency. You should use the `CommonHelper` component here.

**Comment by leleuj** [comment]
+1 with a small change

**Comment by leleuj** [comment]
Good. Thanks

**Pull request merged**